### DISCUSSION TITLE
Redis Object Cache affects the Site Health report on wordpress admin dashboard

### DISCUSSION THREAD

**Discussion by elyograg** [rhubarbgroup/.github#29] [Support (ROC)] [answered]
I went to the wordpress forums for help with an odd issue.

https://wordpress.org/support/topic/site-health-on-dashboard-shows-problems-but-site-health-shows-all-good/#post-17060926

I figured something out ... if the redis object cache is flushed, then the dashboard Site Health report changes, from whatever it was saying before to "No information yet…".  The wordpress forum moderator I am dealing with said this little fact is not a good thing.

There are no actual options in the ROC settings page ... the cache can be flushed and the cache can be disabled.  That's it.

What thoughts do you have about that?

**Comment by tillkruss** [comment, marked as answer]
>There are no actual options in the ROC settings page ... the cache can be flushed and the cache can be disabled. That's it. What thoughts do you have about that?

That's intentional.

WordPress core uses transients (I believe) for storing Site Health checks, and when a persistent object cache like _Redis Object Cache_ is enabled, then those transients will use Redis instead.

**Comment by elyograg** [comment]
OK, so that explains why clearing the cache affects the dashboard info about site health.

I can't do anything to reproduce the original issue, where the dashboard says there are health problems but going to the Site Health page shows no problems.  I just have to wait for it to occur again, and I do not have any idea how long that might take.  And then I will need to disable ROC and wait again, to see if it happens without the plugin.
